portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -75,6 +75,10 @@ definition {
 
 	@priority = "4"
 	test AddMBThreadWithLinkToDocument {
+		property app.server.types = "jboss,tcserver,tomcat,weblogic,websphere,wildfly";
+		property database.types = "db2,hypersonic,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
+		property environment.acceptance = "true";
+
```

Comment:
I feel there's no need to include these tests in environment tester...your reasoning?

Reply:
The browser types that listed in ticket [LPS-120582](https://issues.liferay.com/browse/LPS-120582) are Chrome, IE11. So I added those properties so they can run on IE. If I misunderstand, I will remove those properties.

Reply:
oic, sorry I forgot to delete those. They were meant for Manual Testing scope. Please remove the properties and go ahead and forward. :+1: